source/making_payments/index.html.md.erb

```diff
@@ -121,8 +143,8 @@ Use a GET request against a single payment to find out if you can cancel it:
 
 `GET /v1/payments/paymentId`
 
-If the JSON response body contains a `"cancel"` field in the `_links` object that is not `null`, you
-can cancel the payment. For example:
+If the JSON response body contains a `"cancel"` field in the `_links` object
+that is not `null`, you can cancel the payment. For example:
```

Comment:
Not entirely clear here that it’s the `"cancel"` field that’s not `null`, rather than the `_links` object.

Reply:
Tried to make clearer by putting in parentheses